Services/Mail/classes/Form/class.ilManualPlaceholderInputGUI.php

```diff
@@ -151,7 +124,7 @@ public function insert($a_tpl)
      * @param bool $ajax
      * @return string|void
      */
-    public function render($ajax = false)
+    public function render(bool $ajax = false) : mixed
```

Comment:
The `mixed` type is not supported in PHP 7.4, which we'll have to support as well in ILIAS 8.